src/Command/RestoreProject.php

```diff
@@ -378,55 +403,56 @@ protected function execute(InputInterface $input, OutputInterface $output): int
+                    $configuration->setName($configurationData->name);
                     $components->addConfiguration($configuration);
 
-                    if (isset($configurationData->_versions[0])) {
-                        $restoreVersion = $lastConfigurationVersion = $configurationData->_versions[0];
-                    } else {
-                        $restoreVersion = $configurationData;
-                    }
+                    // update configuration and state
                     $configuration->setChangeDescription(
                         "Configuration {$componentConfiguration["id"]} restored from backup"
                     );
-                    $configuration->setConfiguration($restoreVersion->configuration);
-                    if (isset($restoreVersion->state)) {
-                        $configuration->setState($restoreVersion->state);
+                    $configuration->setConfiguration($configurationData->configuration);
+                    if (isset($configurationData->state)) {
+                        $configuration->setState($configurationData->state);
                     }
                     $components->updateConfiguration($configuration);
 
+                    // create configuration rows
                     if (count($configurationData->rows)) {
                         foreach ($configurationData->rows as $row) {
+                            // create empty row
                             $configurationRow = new ConfigurationRow($configuration);
                             $configurationRow->setRowId($row->id);
                             $components->addConfigurationRow($configurationRow);
+
+                            // update row configuration and state
                             $configurationRow->setConfiguration($row->configuration);
                             $configurationRow->setChangeDescription("Row {$row->id} restored from backup");
+                            if (isset($row->state)) {
+                                $configurationRow->setState($row->state);
+                            }
                             $components->updateConfigurationRow($configurationRow);
                         }
                     }
                 }
+
+                $output->writeln($this->check());
             }
-            $output->writeln($this->check());
         }
 
         $output->writeln("Project successfully restored. Please note what's missing:");
+        $output->writeln(" - linked buckets");
         $output->writeln(" - table snapshots");
         $output->writeln(" - created/modified date of all objects");
         $output->writeln(" - encrypted data (eg. passwords)");
         $output->writeln(" - oauth authorizations");
+        $output->writeln(" - configurations for non-existing components");
+        $output->writeln(" - configurations for components having custom REST API (eg. orchestrator, gooddata-writer)");
```

Comment:
tady by měly bejt vypsaný všechny komponenty, který nejsou restorovaný. zmínka o API (který navíc spíš není REST) je zbytečná, uživatel netuší co/proč tomu brání

Reply:
Nerestorovany komponenty vidis v tom vypisu co jsem screenshotoval. Nebo myslis ze to nestaci?

Reply:
jo jasně, tak to stačí, tak bych upravil jen tu hlášku na `configurations for components having custom API`.